ztag/transforms/ftp.py

```diff
@@ -31,3 +32,40 @@ def _transform_object(self, obj):
         out['timestamp'] = obj['timestamp']
         zout.transformed = out
         return zout
+
+class FTPSTransform(ZGrabTransform):
+    name = "ftp/ftps"
+    port = 21
+    protocol = protocols.FTP
+    subprotocol = protocols.FTP.TLS
+
+    def __init__(self, *args, **kwargs):
+        super(FTPSTransform, self).__init__(*args, **kwargs)
+
+    def _transform_object(self, obj):
+        ftp_banner = obj
+        ftp = Transformable(obj)
+        zout = ZMapTransformOutput()
+        error = ftp['error'].resolve()
+        if error != None:
```

Comment:
If banner is non-null, but we timeout/error/are told there's no support for AuthTLS and AuthSSL, will the error field in the ZGrab output be non-null?